src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    //to store the sentence
+    private static String line;
+    //to keep track of the number of task
+    private static int count = 0;
+    private static Scanner in = new Scanner(System.in);
```

Comment:
Maybe you should consider using a better name for the Scanner variable, probably "scanner".